src/main/java/core/basesyntax/strategy/impl/BalanceOperationService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperationService implements OperationHandler {
```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```
the same for other implementations